src/test/java/me/moodcat/backend/rooms/RoomInstanceTest.java

```diff
@@ -83,4 +117,70 @@ public void whenOneUserSendsMessagesTooFastThrowsException() {
         }
     }
 
+	@Test
+	public void testReplayHistoryOnNoResults() {
+		when(songDAO.findForDistance(room.getVaVector(), RoomInstance.NUMBER_OF_SELECTED_SONGS))
+			.thenReturn(Collections.emptyList());
+
+		assertThat(room.getPlayHistory(), Matchers.empty());
```

Comment:
Are these asserts really necessary? As they test the behaviour of the mocks. Else make them preconditions.